frontend/src/bundles/auth/components/divider/divider.tsx

```diff
@@ -0,0 +1,15 @@
+import clsx from 'clsx';
+
+import styles from './styles.module.scss';
+
+type DividerPayload = {
+    isBlue?: boolean;
```

Comment:
Consider replacing it with `variant` prop and have several variants like 'primary' | 'secondary' etc